### ISSUE TITLE
ThemeRoller: Customizable link color

### ISSUE THREAD

**Issue by scottgonzalez** [jquery/download.jqueryui.com#35]
Currently the jquery ui css overrides the link color specified in your own css. On my project, we need the links to be blue and underlined for accessibility reasons, but the ui css sets it to the same color as the text.

You can override these styles, but in doing so you are creating bug-prone, brittle code.

It'd sure be nice if we could customize the color for plain-text links inside ui widgets using themeroller. That way such hacks could be avoided.

http://bugs.jqueryui.com/ticket/6018

**Comment by Isotab** [comment]
This has been open for 3 years now, are there any plans to do something about it? This is the only part of the css that actually needs touched when you download a theme so it would really be nice if this were added to the ui.

Now since the original poster did not offer a suggestion for what it should look like I will.
- A separate accordion tab just for links since the color/style should be universal
- Three color choices for a:link, a:hover, a:visited. (Maybe another for a:active but I personally do not see that as needed, just my opinion though)
- Size input selection (just like font settings)
- Font weight selection (just like font settings)
- Checkbox for underline
- Another possible suggestion is to make the size/weight/underline options available for each state of the anchor tag.

I think those options should cover most cases of what people would want to customize, if they want to take it even further that is something they should have to go into the css and edit themselves.

**Comment by scottgonzalez** [comment]
I assume you're referring to the `.ui-widget-content a` rule? I'm actually not sure why this rule even exists. We should probably just drop it from the theme.

**Comment by arschmitz** [comment]
I agree this does not really make any sense could probably just drop it and double check through all the demos nothing weird happens or gets broken.
